### PR TITLE
Minor Fixes to documentation

### PR DESCRIPTION
**Pull request by 0xpulkit**

### CODE REVIEW

File: docs/wallet/universal-links.md [hunk 1]
```diff
@@ -269,19 +269,32 @@ const finishUrl = encodeURIComponent("https://my-app.org/finish");
 // Base64 encode the verification request
 const base64EncodedRequest = btoa(JSON.stringify(request));
 
-// URL encode the verification request with the parameters
-const encodedRequestWithParams =(`i_m=${base64EncodedRequest}&back_url=${backUrl}&finish_url=${finishUrl}`);
 
-// Open the Web Wallet URL with the encoded request
-window.open(`https://wallet-dev.privado.id/#${encodedRequestWithParams}`);
+// Configure the Wallet URL(universal link)
+walletUrlWithMessage = `https://wallet-dev.privado.id/#i_m=${base64EncodedRequest}&back_url=${backUrl}&finish_url=${finishUrl}`;
```
Comment (amonsosanz) [inline docs/wallet/universal-links.md:274]: `https://wallet-dev.privado.id`?

File: docs/wallet/universal-links.md [hunk 2]
```diff
@@ -269,19 +269,32 @@ const finishUrl = encodeURIComponent("https://my-app.org/finish");
...
+
+// You can also use the `request_uri` parameter instead of `i_m`. 
+// For that, first define the URL containing the request, and URI encode it.
 
-// You can also use the `request_uri` param instead of `i_m`, For that first define the Url containing the request.
 const requestUrl = encodeURIComponent("https://raw.githubusercontent.com/0xpulkit/Examples_Privado-ID/main/KYCV3.json");
 
-// Configure the Web Wallet URL
+// Configure the Wallet URL using `request_uri` instead of `i_m`
 walletUrlWithRequestUri = `https://wallet-dev.privado.id/#request_uri=${requestUrl}&back_url=${backUrl}&finish_url=${finishUrl}`);
```
Comment (amonsosanz) [inline docs/wallet/universal-links.md:290]: Same. We should always point to production.

File: docs/wallet/universal-links.md (hunk 1 above)
Comment (0xpulkit) [reply docs/wallet/universal-links.md]: fixed it

File: docs/wallet/web-wallet.md [hunk 3]
```diff
@@ -89,6 +89,10 @@ The Web Wallet can be seamlessly integrated into the credential issuance flow by
 
 You can experience a demo of the Web Wallet integration [here](https://web-wallet-demo.privado.id/).
 
+:::caution
+The prior method of integrating the Web Wallet using `https://wallet.privado.id/#base64EncodedData` is now deprecated and will no longer be supported after Release 9.
```
Comment (amonsosanz) [inline docs/wallet/web-wallet.md:93]: Release 8 you mean?

File: docs/wallet/universal-links.md (hunk 2 above)
Comment (0xpulkit) [reply docs/wallet/universal-links.md]: Fixed it

File: docs/wallet/universal-links.md [hunk 4]
```diff
@@ -269,19 +269,32 @@ const finishUrl = encodeURIComponent("https://my-app.org/finish");
 // Base64 encode the verification request
 const base64EncodedRequest = btoa(JSON.stringify(request));
 
-// URL encode the verification request with the parameters
-const encodedRequestWithParams =(`i_m=${base64EncodedRequest}&back_url=${backUrl}&finish_url=${finishUrl}`);
 
-// Open the Web Wallet URL with the encoded request
-window.open(`https://wallet-dev.privado.id/#${encodedRequestWithParams}`);
+// Configure the Wallet URL(universal link)
```
Comment (amonsosanz) [inline docs/wallet/universal-links.md:273]: missing space

> `URL(uni`

File: docs/wallet/web-wallet.md (hunk 3 above)
Comment (0xpulkit) [reply docs/wallet/web-wallet.md:93]: Nope release 9, we will support it for the time being.
Agreed with Sasha on this

File: docs/wallet/universal-links.md (hunk 4 above)
Comment (amonsosanz) [reply docs/wallet/universal-links.md:273]: Also, why removing `Web`? We should always refer to `Web Wallet`

Comment (0xpulkit) [reply docs/wallet/universal-links.md]: Fixed

File: docs/wallet/web-wallet.md (hunk 3 above)
Comment (amonsosanz) [reply docs/wallet/web-wallet.md:93]: Ahh sorry I misread, you say we are removing support for the old format in the next release. Cool.

File: docs/wallet/universal-links.md (hunk 4 above)
Comment (amonsosanz) [reply docs/wallet/universal-links.md]: Ok. I guess...

**Review by amonsosanz — APPROVED** [review]

**Pull request merged**